composer/callbacks/speed_monitor.py

```diff
@@ -41,19 +42,21 @@ class SpeedMonitor(Callback):
     The training throughput is logged by the :class:`.Logger` to the following keys as
     described below.
 
-    +----------------------------------+-------------------------------------------------------------+
-    | Key                              | Logged data                                                 |
-    +==================================+=============================================================+
-    |                                  | Rolling average (over ``window_size`` most recent           |
-    | ``throughput/samples_per_sec``   | batches) of the number of samples processed per second      |
-    |                                  |                                                             |
-    +----------------------------------+-------------------------------------------------------------+
-    | ``wall_clock/train``             | Total elapsed training time                                 |
-    +----------------------------------+-------------------------------------------------------------+
-    | ``wall_clock/val``               | Total elapsed validation time                               |
-    +----------------------------------+-------------------------------------------------------------+
-    | ``wall_clock/total``             | Total elapsed time (wall_clock/train + wall_clock/val)      |
-    +----------------------------------+-------------------------------------------------------------+
+    +-----------------------------------+---------------------------------------------------------+
+    | Key                               | Logged data                                             |
+    +===================================+=========================================================+
+    |                                   | Rolling average (over ``window_size`` most recent       |
+    | ``throughput/samples_per_sec``    | batches) of the number of samples processed per second  |
+    |                                   |                                                         |
+    +-----------------------------------+---------------------------------------------------------+
+    | ``wall_clock/train``              | Total elapsed training time                             |
+    +-----------------------------------+---------------------------------------------------------+
+    | ``wall_clock/val``                | Total elapsed validation time                           |
+    +-----------------------------------+---------------------------------------------------------+
+    | ``wall_clock/total``              | Total elapsed time (wall_clock/train + wall_clock/val)  |
+    +-----------------------------------+---------------------------------------------------------+
+    | ``wall_clock/remaining_estimate`` | Estimated time to completion                            |
```

Comment:
Given the hairiness of this estimate. Please spell out exactly how the estimate is computed.